src/ResXManager.View/Visuals/TranslationItem.cs

```diff
@@ -44,14 +44,25 @@ public TranslationItem(ResourceTableEntry entry, string source, CultureKey targe
             set => _translation = value;
         }
 
-        public bool Apply(string? prefix)
+        public bool UpdateValue(string? prefix)
         {
             if (!_entry.CanEdit(TargetCulture))
                 return false;
 
             return _entry.Values.SetValue(TargetCulture, prefix + Translation);
         }
 
+        public bool UpdateComment(string? prefix)
+        {
+            var neutralLanguageComment = _entry.Comment;
+
+            if (neutralLanguageComment == null)
+                return false;
+
+            _entry.Comment = prefix + neutralLanguageComment;

```

Comment:
Shouldn't that check if the prefix is already present? Since you add this to the neutral comment, you will end up with multiple prefixes when applying translations for multiple languages.

Reply:
Yes it should, good call.